R/as_polars_lf.R

```diff
@@ -19,7 +19,15 @@ as_polars_lf <- function(x, ...) {
 #' @rdname as_polars_lf
 #' @export
 as_polars_lf.default <- function(x, ...) {
-  as_polars_df(x, ...)$lazy()
+  try_fetch(
+    as_polars_df(x, ...),
+    error = function(cnd) {
+      abort(
+        "Failed to create a polars DataFrame.",
```

Comment:
```suggestion
        "Failed to create a polars LazyFrame.",
```
Also why not put the `$lazy()` in `try_fetch()`?

Reply:
I would like to isolate here whether the creating a DataFrame failed to create or failed in lazy.

Reply:
Yeah but then we're back to the same issue where the user calls `as_polars_lf()` and sees `"Failed to create a polars DataFrame"`

Reply:
Also, I think the `parent = cnd` will show the error originating from `as_polars_df()`. So the user will still see in the trace that the source of the error comes from the creation of the polars DataFrame, but the main message should be "Failed to create a polars LazyFrame".